server/scsynth/SC_PortAudio.cpp

```diff
@@ -293,26 +293,23 @@ int SC_PortAudioDriver::PortAudioCallback( const void *input, void *output,
 	return paContinue;
 }
 
-void SC_PortAudioDriver::GetPaDeviceFromName(const char* device, int* mInOut) {
+void SC_PortAudioDriver::GetPaDeviceFromName(const char* device, int* mInOut, bool isInput) {
```

Comment:
I would recommend creating an enum class for this parameter:

```cpp
enum class IOType {
    Input,
    Ouptut
};
```

so that the calls to `GetPaDeviceFromName` below make more sense in context. then this signature becomes

`void SC_PortAudioDriver::GetPaDeviceFromName(const char* device, int* mInOut, IOType ioType)`